### ISSUE TITLE
Rubocop lints should source while preserving line-endings

### ISSUE THREAD

**Issue by agross** [brigade/haml-lint#167]
I found that haml-lint does not preserve line-endings while extracting Ruby source code on Windows. In a LF file this piece of code is reported as `some.haml:1 [W] RuboCop: Style/EndOfLine: Carriage return character detected.`

some.haml:
```haml
!!!
%html{ lang: language_code_of(@item) }
```

I haven't looked at RubyExtractor, but opening the temporary file in binmode seems to fix the problem.

**Comment by agross** [comment]
Further investigation revealed that RubyExtractor actually uses `\n` to join the lines, but these are translated to `\r\n` while writing to the temp file that rubocop processes: https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/rubocop.rb#L36

Add `f.binmode` before `f.write` fixes the issue entirely. I'm not sure how to write a test and check whether the temp file receives the correct line endings. The usefulness of such a test is also questionable since you run yours builds on Travis/Linux.

**Comment by sds** [comment]
Addressed in #168. Thanks for the report!
